src/Microsoft.Data/BaseColumn.Computations.cs

```diff
@@ -33,41 +33,81 @@ public virtual void CumulativeMax()
             throw new NotImplementedException();
         }
 
+        public virtual void CumulativeMax(IEnumerable<long> rows)
```

Comment:
Is there a better name for this parameter other than `rows`? These aren't really the rows themselves - these are the rowIndices, right?